content/help/faq/general/how-to-join-slack.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How can I join Istio Slack workspace?
+weight: 180
+---
+
+Istio Slack is restricted to Istio Developers. See the [Community Roles](https://github.com/istio/community/blob/master/ROLES.md)
+guide, specifically the **Member** role. In this page you can find instructions on how to apply for Istio membership,
```

Comment:
could you link to the member directly like:  https://github.com/istio/community/blob/master/ROLES.md#member